deploy/charts/kube-oidc-proxy/templates/deployment.yaml

```diff
@@ -20,6 +20,9 @@ spec:
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
       serviceAccountName: {{ include "kube-oidc-proxy.fullname" . }}
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 65534
```

Comment:
These should be piped through from values, rather than hardcoded here.

Reply:
@JoshVanL  - How does that look ? :)